modin/data_management/utils.py

```diff
@@ -38,7 +38,7 @@ def get_default_chunksize(length, num_splits):
     )
 
 
-def compute_chunksize(df, num_splits, default_block_size=32, axis=None):
+def compute_chunksize(df, num_splits, default_block_size=1, axis=None):
```

Comment:
@devin-petersohn, since we have #3662 that is going to resolve the issue with metadata management, let's create an issue on finer-grained partitioning and close it with this PR. What do you think?